playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -44,6 +44,23 @@
   roles:
   - openshift_master_facts
 
+- name: configure vsphere svc account
+  include_role:
+    name: openshift_cloud_provider
+    tasks_from: vsphere-svc
+  when:
+  - openshift_cloudprovider_kind in 'vsphere'
+  - openshift_version | version_compare('3.9', '>=')
+  - inventory_hostname == openshift_master_hosts[0]
+
+- name: update vsphere provider master config
```

Comment:
Then since this should be run on all masters, move this down to after https://github.com/openshift/openshift-ansible/pull/7096/files#diff-6e1f944c172a66b8294fa8cc2b081a97R82